src/com/sap/piper/MapUtils.groovy

```diff
@@ -63,6 +63,19 @@ class MapUtils implements Serializable {
         m.putAll(updates)
     }
 
+    static private def getByPath(Map m, def key) {
+        if (m == null) return null
+
+        List path = key instanceof String ? key.tokenize('/') : key
```

Comment:
What about `in CharSequence` rather than explict check for `String`. With the check for `CharSequence` we cover also `GString`s.

Reply:
```suggestion
        List path = key in CharSequence ? key.tokenize('/') : key
```